tasks/managed/get-push-koji-secret/get-push-koji-secret.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-secrets
+spec:
+  params:
+    - name: dataPath
+  steps:
+    - name: get-secrets
+      image: quay.io/redhat-appstudio/release-service-utils
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)/data.json"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No data JSON was provided."
+            exit 1
+        fi
+
+        # Put the .pushOptions.pushKeytab into the secret that can be used on other task
+        jq -jr '.pushOptions.pushKeytab.secret' "$DATA_FILE" > '$(results.secret.path)'
```

Comment:
Maybe use `.pushOptions.pushKeytabSecret` - no need for another level? (It will also need to be updated in the comment above.) Also, this should be added to `schema/dataKeys.json`

Reply:
The data structure I defined is like:
"pushKeytab": {
                    "name": "test.keytab",
                    "principle": "test@test.com",
                    "secret": "test-secrets"
                  },
So maybe better to keep the nested levels here.

Reply:
That seems unnecessarily verbose. Do we really need to keep all of these in the RPA? Can't we just have a secret name and the rest be saved in the secret, one key per each?

Reply:
I can see the principle now is stored as a base64_keytab file, so it's not the original one, we'd better to keep the flexibility.

Reply:
I missed the last point you mention, sorry about that, @mmalina .
@johnbieren thanks for your reminding.